.gitignore

```diff
@@ -346,7 +346,7 @@ __pycache__/
 
 # Distribution / packaging
 .Python
-build/
+# build/
```

Comment:
This line should remain in the gitignore

Reply:
Done